### PR TITLE
Added Usage Examples for Graph Convolutional Layers 

### PR DESCRIPTION
**Pull request by rbSparky**
Implements Issue #395

### CODE REVIEW

**Comment by rbSparky** [comment]
Will make this PR ready for merge when more examples are added

**Comment by rbSparky** [comment]
Ready for merge?

Done for all except TransformerConv
where this did not work:

```julia
# create data
s = [1,1,2,3]
t = [2,3,1,1]
in_channel = 3
out_channel = 5
ein = 2
heads = 3
g = GNNGraph(s, t)

# create layer
l = TransformerConv(in_channel * heads => in_channel; heads, add_self_loops = true,
root_weight = false, ff_channels = 10, skip_connection = true,
batch_norm = false)

# edge features
e = randn(ein, length(s))

# forward pass
y = l(g, x, e) 
```

giving:
`ERROR: DimensionMismatch: layer Dense(9 => 9) expects size(input, 1) == 9, but got 3×3 Matrix{Float64}`
Let me know what can be done to fix this

File: src/layers/conv.jl [hunk 1]
```diff
@@ -187,6 +187,23 @@ with ``\hat{L}`` the [`scaled_laplacian`](@ref).
...
+
+# Examples
+
+```julia
+
+# create data
+s = [1,1,2,3]
+t = [2,3,1,1]
+g = GNNGraph(s, t)
+x = randn(3, g.num_nodes)
```
Comment (CarloLucibello) [inline src/layers/conv.jl:199]:
```suggestion
x = randn(Float32, 3, g.num_nodes)
```
This should be changed everywhere.

File: src/layers/conv.jl [hunk 2]
```diff
@@ -187,6 +187,23 @@ with ``\hat{L}`` the [`scaled_laplacian`](@ref).
 - `k`: The order of Chebyshev polynomial.
 - `bias`: Add learnable bias.
 - `init`: Weights' initializer.
+
+# Examples
+
+```julia
+
```
Comment (CarloLucibello) [inline src/layers/conv.jl:194]:
```suggestion
```

**Comment by rbSparky** [comment]
Done

File: src/layers/conv.jl (hunk 1 above)
Comment (rbSparky) [reply src/layers/conv.jl]: Done

**Pull request merged**